llvm/test/CodeGen/X86/canonicalize-constants.ll

```diff
@@ -0,0 +1,210 @@
+; RUN: llc --mcpu=sapphirerapids -mtriple=x86_64 < %s | FileCheck %s
```

Comment:
Don't use a specific CPU for generic tests - do multiple RUNs with either -mattr SSE/AVX1/AVX2/AVX512 style or -mcpu x86-64/x86-64-v2/x86-64-v3/x86-64-v4

Reply:
Noted. Will make changes.